nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java

```diff
@@ -143,4 +146,14 @@ public boolean await(Duration duration) throws InterruptedException {
         }
         return true;
     }
+
+    @Override
+    public List<BulkStatus> getStatuses(String username) {
+        KeyValueStoreProvider kv = (KeyValueStoreProvider) getKvStore();
+        return kv.keyStream()
+                 .filter(key -> key.endsWith(COMMAND)
```

Comment:
Note the comment on the `keyStream` Javadoc: `This operation may be slow and should only be used for management or debug purposes`
So this is going to be slow if there are lots of keys. Is this something that we expect? If not, it's worth explaining why.

Reply:
This is something we have in mind, and for now the only way to do it. Also as we want to have a specific store for bulk actions, we shouldn't end up with lots of keys.